README.md

```diff
@@ -15,6 +15,9 @@ Another Android Monero Wallet for Monero
 - Advanced users can copy over synced wallet files (all files) onto sdcard in directory Monerujo (created first time App is started)
 - See the [FAQ](doc/FAQ.md)
 
+## Translations
+help us translating Monerujo! you can find instructions [On Taiga](https://taiga.getmonero.org/project/erciccione-monero-localization/wiki/monerujo), but if you need help/support open an issue or contact the Localization Workgroup, you can find us on the freenode channel `#monero-translations`, also relayed on [MatterMost](https://mattermost.getmonero.org/monero/channels/monero-translations), and matrix/riot.
```

Comment:
Small changes:

Help us translating Monerujo! You can find instructions [on Taiga](https://taiga.getmonero.org/project/erciccione-monero-localization/wiki/monerujo), and if you need help/support, open an issue or contact the Localization Workgroup. You can find us on the freenode channel `#monero-translations`, also relayed on [Mattermost](https://mattermost.getmonero.org/monero/channels/monero-translations) and matrix/riot.

Reply:
i missed this - added your suggestions it their own pr. thanks.